### PR TITLE
Navbar Updated new PR

### PR DESCRIPTION
**Pull request by srrathi**
## Related Issue

- Info about the related issue
- New Navbar according to the required design is updated

Closes: #331 

### Describe the changes you've made
I've made a new navbar using bootstrap and react-bootstrap according to the design we required.

Give a clear description what modifications you have made
removed the previous navbar and added the new navbar

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot |
nvold
 updated screenshot |
nvnew

### CODE REVIEW

**Comment by msk4862** [comment]
@srrathi add screenshot for mobile screen too.

**Review by msk4862 — COMMENTED** [review]
@srrathi add screenshot for mobile screen too.

**Comment by srrathi** [comment]
> @srrathi add screenshot for mobile screen too.

sure @msk4862 
image
image
image

File: src/components/Navbar/header.jsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+          <Navbar.Collapse id="basic-navbar-nav">
+            <Nav className="mx-auto">
+              <Nav.Link className="px-lg-5">
+                <Link className="text-white text-decoration-none" href="/feed">
+                  <p className="text-white text-decoration-none my-lg-3">
+                    FEED
+                  </p>
+                </Link>
+                {router.pathname === '/feed' && (
+                  <hr
```
Comment (msk4862) [inline src/components/Navbar/header.jsx:64]: @srrathi Maybe we can create a separate component for `hr`.  As it is used in multiple places.

File: src/components/Navbar/header.jsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+                </div>
+                <div style={{ width: '100px' }}>
+                  {User !== null && <strong> {User.name} </strong>}
+                </div>
+              </Dropdown.Toggle>
+
+              <Dropdown.Menu
+                style={{ left: '50%', transform: 'translateX(-50%)' }}
+                className="mt-2">
+                <Dropdown.Item href="#/action-1">
```
Comment (msk4862) [inline src/components/Navbar/header.jsx:176]: Screenshot from 2021-05-04 11-55-43
Remove this blue background on click of Menu item.

Screenshot from 2021-05-04 11-48-06
Also remove its grey highlighting on hover.

**Review by msk4862 — CHANGES_REQUESTED** [review]

File: src/components/Navbar/header.jsx (hunk 2 above)
Comment (srrathi) [reply src/components/Navbar/header.jsx:176]: Okay @msk4862 I'll do that by EOD.

File: src/components/Navbar/header.jsx (hunk 1 above)
Comment (srrathi) [reply src/components/Navbar/header.jsx:64]: Yeah sure. I'll do that.

File: src/components/Navbar/header.jsx (hunk 2 above)
Comment (srrathi) [reply src/components/Navbar/header.jsx]: > Screenshot from 2021-05-04 11-55-43
> Remove this blue background on click of Menu item.
> 
> Screenshot from 2021-05-04 11-48-06
> Also remove its grey highlighting on hover.

@msk4862 I've made all the changes you requested please check.

**Comment by msk4862** [comment]
@srrathi have you tested again? Because Dropdown item bg is still blue on click and grey on hover.
You can use `.dropdown-item:active` and `.dropdown-item:hover` respectively to override styles.

**Comment by srrathi** [comment]
> @srrathi have you tested again? Because Dropdown item bg is still blue on click and grey on hover.
> You can use `.dropdown-item:active` and `.dropdown-item:hover` respectively to override s

> @srrathi have you tested again? Because Dropdown item bg is still blue on click and grey on hover.
> You can use `.dropdown-item:active` and `.dropdown-item:hover` respectively to override styles.

@msk4862  I'm sorry the changes which i made worked earlier but now I've actually solved it. you can check now.

**Comment by msk4862** [comment]
Looks good now :+1:

**Review by aryasoni98 — APPROVED** [review]

**Pull request merged**